.gitattributes

```diff
@@ -1,5 +1,7 @@
 * text=auto
 
+*.bat text eol=crlf
+*.patch -text
```

Comment:
Thanks @jschueller. It would be interesting to see how those patches are being changed. The re-rendering bot simply stages all changes, which is not required and typical re-rendering would not do. So it should be a simple matter to stop it from doing that. However, I wouldn't worry about it right now as the re-rendering bot is out of commission ATM.